spec/models/date_of_birth_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe DateOfBirth do
+  subject(:date_of_birth) { DateOfBirth.new({'date_of_birth(3i)' => '1', 'date_of_birth(2i)' => '12', 'date_of_birth(1i)' => '2001'}) }
```

Comment:
Since we're not using implicit testing here can we make these `let` instead of `subject`?

Reply:
yep